packages/usa-time-picker/src/test/time-picker-regex.spec.js

```diff
@@ -1,7 +1,35 @@
 const assert = require("assert");
-const { generateDynamicRegExp } = require("../../../usa-combo-box/src/index");
```

Comment:
@kodyjackson the component still relies on combo box. Is there a particular reason why we removed this?

Reply:
The time-picker relied on an earlier version of `generateDynamicRegExp`, which matched strings by checking if any character from the list was present. The updated version, however, creates a regular expression that matches the starting characters of the list elements with the search query.